src/popup/router/components/SidebarMenu.vue

```diff
@@ -92,27 +92,20 @@ export default {
   mixins: [removeAccountMixin],
   components: { Close, Arrow, UserAvatar },
   computed: mapGetters(['account', 'activeAccountName']),
-  data: () => ({ showSettingsDropdown: false, showTokensDropdown: false, balances: null }),
-  watch: {
-    showTokensDropdown(val) {
```

Comment:
This should be present for the FT functionality I think.

Reply:
Well, I'm proposing to introduce it in the same time as fungible tokens support.